src/components/ImageView/index.native.js

```diff
@@ -102,148 +115,123 @@ class ImageView extends PureComponent {
      *
      * @param {Object} nativeEvent
      */
-    configureImageZoom({nativeEvent}) {
-        let imageWidth = nativeEvent.width;
-        let imageHeight = nativeEvent.height;
-        const containerWidth = Math.round(this.props.windowWidth);
-        const containerHeight = Math.round(this.state.containerHeight ? this.state.containerHeight : this.props.windowHeight);
+    const configureImageZoom = ({nativeEvent}) => {
+        let imgWidth = nativeEvent.width;
+        let imgHeight = nativeEvent.height;
```

Comment:
For me it is a little bit confused, because we already have `imageWidth` and `imageHeight` in the states. Let's rename it like: `imageZoomWidth` and `imageZoomHeight`.